test/integration/resultsPagePlace.js

```diff
@@ -131,6 +131,9 @@ describe('The place results page', () => {
         iExpect.htmlWith200Status(err, res);
         const $ = cheerio.load(res.text);
         expect($('.results__header--none').text()).to.be.equal(`We can't find '${noResultsTerm}'`);
+        expect($('.results__none-content').text()).to
+          .contain('If the place you searched for is in England, you could:');
+        expect($('.results-none-nearby').length).to.be.equal(0);
```

Comment:
Can this be extended to include this test please - https://github.com/nhsuk/connecting-to-services/pull/393/commits/951b40d7f1af2bd2bf57bf0bb45deaa681667cce#diff-d47eaffdfc3f1a1634cee7b96469fa02R175